packages/tupaia-web/src/components/Modal.tsx

```diff
@@ -53,17 +53,13 @@ export const Modal = ({ children, isOpen, onClose }: ModalProps) => {
   const theme = useTheme();
   const fullScreen = useMediaQuery(theme.breakpoints.down('sm'));
   return (
-    <Dialog
-      open={isOpen}
-      onClose={onClose}
-      PaperComponent={Paper}
-      fullScreen={fullScreen}
-      disablePortal
-    >
-      <CloseButton onClick={onClose} color="default">
-        <CloseIcon />
-      </CloseButton>
-      <Wrapper id="overlay-wrapper">{children}</Wrapper>
+    <Dialog open={isOpen} onClose={onClose} PaperComponent={Paper} fullScreen={fullScreen}>
+      <Wrapper id="overlay-wrapper">
+        <CloseButton onClick={onClose} color="default">
```

Comment:
Looks like the CloseButton has been moved to inside `Wrapper` - any particular reason?